src/wavm/mpi.cpp

```diff
@@ -18,10 +18,10 @@ using namespace faabric::scheduler;
 using namespace WAVM;
 
 #define MPI_FUNC(str)                                                          \
-    SPDLOG_DEBUG("MPI-{} {}", executingContext.getRank(), str);
+    SPDLOG_TRACE("MPI-{} {}", executingContext.getRank(), str);
```

Comment:
These were printing out so much logging information in the dist tests that it was difficult to debug.